gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/cloning/Cloning.java

```diff
@@ -35,45 +36,55 @@
 import org.gradoop.flink.model.impl.operators.cloning.functions.Value0Of2ToId;
 
 /**
- * Creates a copy of the logical graph with new ids for the graph head,
- * vertices and edges.
+ * Creates a copy of the logical graph with new ids for the graph head, vertices and edges.
```

Comment:
Creates a copy of the graph instance with new ids for the graph head, vertices and edges.